hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -401,17 +403,18 @@ cdef class PureMarketMakingStrategy(StrategyBase):
     cdef c_process_market_pair(self, object market_pair, list active_orders):
         cdef:
             double last_trade_price
-            MarketBase maker_market
+            MarketBase maker_market = market_pair.maker_market
             OrderBook maker_order_book
             bint is_buy
             double current_timestamp = self._current_timestamp
+            str maker_name = maker_market.__class__.__name__
```

Comment:
You should use the `.name` property instead.

Reply:
changed